src/applications/vaos/utils/calendar.js

```diff
@@ -1,9 +1,38 @@
+import moment from 'moment';
+
+const DEFAULT_MAX_DAYS_AHEAD = 90;
+
 function pad(num, size) {
   let s = num.toString();
   while (s.length < size) s = `0${s}`;
   return s;
 }
 
+export function getMaxMonth(maxDate, startMonth) {
+  const defaultMaxMonth = moment()
+    .add(DEFAULT_MAX_DAYS_AHEAD, 'days')
+    .format('YYYY-MM');
+
+  // If provided start month is beyond our default, set that month as max month
+  if (startMonth && startMonth > defaultMaxMonth) {
```

Comment:
This seems like it would bypass the max days ahead limit?

Reply:
Yeah, this is a little confusing but in the case of direct schedule, `startMonth` is determined by the preferred date that the user selects on the previous page.  I put this in place so that if the user selects a date beyond the `DEFAULT_MAX_DAYS_AHEAD` ahead, we would still display a calendar.